test/metabase/pulse/render/common_test.clj

```diff
@@ -29,29 +31,48 @@
         (is (= "$12,345.54" (fmt {::mb.viz/currency "USD"}))))
       (testing "Other currencies"
         (is (= "AED12,345.54" (fmt {::mb.viz/currency "AED"})))
-        (is (= "₡12,345.54" (fmt {::mb.viz/currency "CRC"})))
+        (is (= "Af12,346" (fmt {::mb.viz/currency "AFN"})))
```

Comment:
add a case where currency specifies a number of decimals but we explicitly pass in requested number of decimals. possibly do this for an integral number as well that is not currency. (testing that explicit decimal places are always honored)